yt/frontends/ramses/fields.py

```diff
@@ -90,7 +91,7 @@ class RAMSESFieldInfo(FieldInfoContainer):
         ("x-acceleration", (ra_units, ["acceleration_x"], None)),
         ("y-acceleration", (ra_units, ["acceleration_y"], None)),
         ("z-acceleration", (ra_units, ["acceleration_z"], None)),
-        ("Potential", (ener_units, ["potential"], None)),
+        ("Potential", (spec_ener_units, ["potential"], None)),
```

Comment:
Is this maybe supposed to be in yt-project/yt#3160?

Reply:
Oh, my bad! It should indeed be there.